docs/future_improvements.md

```diff
@@ -1,3 +1,76 @@
+# Open Auctions
+
+The most widely valuable use case for auctions is the Open Auction, which allows anyone to participate in the auction as bidders.
+
+## Problem Statement (Persistent Deposit Sufficiency)
+
+The chief difficulty with running open auctions is ensuring the bidder deposit is sufficient and remains until the bidder withdraws from the auction.
+
+In private auctions we can easily bypass this problem by offloading the responsibility to the seller, who is both incentivized to ensure the deposit is sufficient, and a trusted party in regards to the auction.
+
+Public auctions must solve this same problem without the involvement of the seller after the initial announcing of the auction. In order to do this, we will rely on the other class of trusted parties in the auction protocol - the hydra delegates.
+
+## Solution
+
+### L1
+
+When a prospective bidder places a deposit, they will be able to immediately bid on L1 by referencing the deposit they placed.
+
+These deposits are stored with datums which include the following information: 
+```haskell
+data PBidderInfo (s :: S)
+  = PBidderInfo
+      ( Term
+          s
+          ( PDataRecord
+              '[ "biBidderAddress" ':= PAddress
+               , "biBidderVk" ':= PByteString
+               ]
+          )
+      )
+  deriving stock (Generic)
+  deriving anyclass (PlutusType, PIsData, PDataFields, PShow, PEq)
+```
+
+To allow for public auctions instead of private auctions, the standing bid validator must be changed to allow bids to be placed by referencing a datum which: 
+- includes a matching address
+- includes a sufficient deposit 
+- is stored at the correct address
+
+### L2
+
+On L2, the hydra delegates are expected to maintain a list of valid deposits in the form of a list of `Address`es which can be similarly referenced:
+- When the auction is first moved to L2, the list is created with all sufficient & valid deposits currently placed at the auction's bidder deposit validator.
+- Whenever a new participant that is not currently in that list attempts to place a bid on L2 the delegate through which that bid is being placed should also update the list of valid deposits to include all new deposits that have been placed since the last update.
```

Comment:
Does it mean we need incremental commit to support it?

Reply:
Just a datum, yeah. I went with this design specifically so we wouldn't need incremental commit.